quantum/audio/song_list.h

```diff
@@ -503,6 +502,17 @@
     W__NOTE(_C5),      \
     W__NOTE(_BF4),
 
+#define KATAMARI_ROLLING_STAR \
```

Comment:
For now, could this be left out?

Reply:
Should I put this in another PR?

Reply:
Moved to https://github.com/qmk/qmk_firmware/pull/5854